pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -158,6 +158,33 @@ func (k *Kubernetes) InitSvc(name string, service kobject.ServiceConfig) *api.Se
 	return svc
 }
 
+// InitConfigMap initialized a ConfigMap object
+func (k *Kubernetes) InitConfigMap(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions, envFile string) *api.ConfigMap {
+
+	envs, err := GetEnvsFromFile(envFile, opt)
+	if err != nil {
+		log.Fatalf("Unable to retrieve env file: %s", err)
+	}
+
+	// Remove root pathing
+	// replace all other slashes / preiods
+	envName := FormatEnvName(envFile)
+
+	// In order to differentiate files, we append to the name and remove '.env' if applicate from the file name
+	configMap := &api.ConfigMap{
+		TypeMeta: unversioned.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: api.ObjectMeta{
+			Name: envName,
```

Comment:
Can this be named like other objects kompose creates? `appName - maifest type.yaml`

Here also we can use same format, just add `appName` before `envName` ?

Reply:
so `appName-configmap-env`?

Reply:
@cdrage this sounds good!